src/cpcli/commands/testcase.py

```diff
@@ -31,7 +31,7 @@ def add_options(self, parser: ArgumentParser) -> None:
             help='Add a new custom test case'
         )
 
-    def run(self, args: Namespace, scraper: Scraper) -> None:
+    def run(self, args: Namespace, scraper: Runner) -> None:
```

Comment:
```suggestion
    def run(self, args: Namespace, runner: Runner) -> None:
```

This should be renamed as well? (same for all the other occurences)